IRremoteESP8266.cpp

```diff
@@ -216,25 +216,28 @@ void IRsend::sendNEC (unsigned long data, int nbits, unsigned int repeat) {
   }
 }
 
-void IRsend::sendLG (unsigned long data, int nbits) {
+void IRsend::sendLG (unsigned long data, int nbits, unsigned int repeat) {
+  // Args:
+  //   data:   The contents of the command you want to send.
+  //   nbits:  The bit size of the command being sent.
+  //   repeat: The number of times you want the command to be repeated.
+
```

Comment:
we can start have function doc if needed and usually for ide to detect function doc it should be before the function signature. besides that looks good.